src/ssh.c

```diff
@@ -840,6 +840,24 @@ int wolfSSH_connect(WOLFSSH* ssh)
                     return WS_FATAL_ERROR;
                 }
             }
+
+            while (ssh->serverState == SERVER_USERAUTH_ACCEPT_KEYBOARD) {
+                if ( (ssh->error = SendUserAuthKeyboardResponse(ssh)) <
+                    WS_SUCCESS) {
+                    WLOG(WS_LOG_DEBUG, connectError, "CLIENT_USERAUTH_SENT",
+                        ssh->error);
+                    return WS_FATAL_ERROR;
+                }
+                ssh->serverState = SERVER_USERAUTH_ACCEPT_KEYBOARD_NEXT;
+                while ((ssh->serverState < SERVER_USERAUTH_ACCEPT_KEYBOARD_DONE) && (ssh->serverState != SERVER_USERAUTH_ACCEPT_KEYBOARD) && (ssh->serverState != SERVER_USERAUTH_ACCEPT_DONE)) {
```

Comment:
please keep the column length to 80 characters or less

Reply:
Yikes, not sure how I managed that, I have a marker for it in my IDE. Thanks.